### DISCUSSION TITLE
SWUpdate in Debos

### DISCUSSION THREAD

**Discussion by openedev** [go-debos/debos#568] [Q&A]
Does anyone use swupdate in debos? Look like OSTree is default supported updater. Any information or sample use case documentation on SWUpdate integration in Debos?

**Comment by obbardc** [comment]
Unfortunately we don't yet have an example, but would be willing to add a community example to go-debos/debos-recipes

**Comment by openedev** [comment]
Tried on target seems working, but while integrating on debos build the package installation of swupdate is using default defconfig (which required extra certificates) which has all swupdate features but usually this is configurable in buildroot, yocto. Any idea how to resolve this? 

One way is to build swupdate in host with specific defconfig and create .deb and intigrate into debos build but it might not be a feasible solution as it require a seperate swupdate build to maintain.

**Comment by obbardc** [comment]
`swupdate` package in debian should have enough features enabled to be generic for most cases.

`swugenerator` package in debian can also be used to build swupdate bundles.

I guess we should close this and have an issue under https://github.com/go-debos/debos-recipes to create an example recipe to integrate `swupdate` ?
